UI/Layers/LayerConfiguration/Misc/LayerCompositionConfigurationWindow.gd

```diff
@@ -74,16 +71,10 @@ func _on_confirm():
 	
 	hide()
 
-# FIXME: Is this still necessary?
-#func resize(add: Vector2):
-#	minimum_size.y = min_size.y + add.y
-#	minimum_size.x = max(add.x, min_size.x) 
-#	size = minimum_size
-
 
 func _add_types():
 	var idx = 0
-	for type in LayerComposition.RenderType:
+	for type in LayerComposition.RENDER_INFOS.keys():
```

Comment:
Does this work? I feel like committing (i.e. pressing ok) might cause an error when loading a new layer